README.md

```diff
@@ -1,129 +1,141 @@
 # Ziti
+
```

Comment:
in the future, while i can (and will) turn off whitespace formatting changes, i'd really prefer changes to content not accompany changes to whitespace